bundle-workflow/.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+default_stages: [commit]
+default_language_version:
+  python: python3.7
+repos:
+- repo: local
+  hooks:
```

Comment:
Should we also specify `files: ^bundle-workflow/` to avoid running the hook on everything else?

Reply:
Although I have made the entry to `bundle-workflow` but yeah adding `files` will be a better idea on top of that